frontend/src/store/actions/addMoviesAction.js

```diff
@@ -1,18 +1,18 @@
-import axios from "../axiosHelper";
+import axios from "../../helpers/axiosHelper";
+import { ADD_MOVIE } from "./constants";
 
 export const addMovies = (movie) => async (dispatch) => {
   const moviesData = await axios.post(`/movies`, {
     ...movie,
     genres: [movie.genres],
-    release_date: "2016-03-11",
+    release_date: movie.release_date,
+    // release_date: "2016-03-11",
     runtime: +movie.runtime,
     date: undefined,
```

Comment:
you don't need to explicitly do that

Reply:
this is updated in my code not sure why it hasnt removed that here